### PR TITLE
Clean .minikube dir and delete any running minikube-windows-amd64.exe…

### PR DESCRIPTION
**Pull request by aaron-prindle**
… processes on start of integration tests

### CODE REVIEW

File: hack/jenkins/windows_integration_test_hyperv.ps1 [hunk 1]
```diff
@@ -17,6 +17,9 @@ gsutil.cmd cp gs://minikube-builds/$env:MINIKUBE_LOCATION/minikube-windows-amd64
 gsutil.cmd cp gs://minikube-builds/$env:MINIKUBE_LOCATION/e2e-windows-amd64.exe out/
 gsutil.cmd cp -r gs://minikube-builds/$env:MINIKUBE_LOCATION/testdata .
 
+
+taskkill /im minikube-windows-amd64.exe /f
+Remove-Item -Recurse -Force C:\Users\jenkins\.minikube
 ./out/minikube-windows-amd64.exe delete
```
Comment (r2d4) [inline hack/jenkins/windows_integration_test_hyperv.ps1]: Why not remove the folder after minikube deletes and skip the `taskkill`? Why do we need to taskkill?

Comment (aaron-prindle) [inline hack/jenkins/windows_integration_test_hyperv.ps1]: There were hanging minikube-windows-amd64.exe processes when I investigated jenkins errors.  To remove the files in .minikube, these processes had the be killed.  I do not see any negative side effects of doing this.

Comment (aaron-prindle) [inline hack/jenkins/windows_integration_test_hyperv.ps1]: I have removed the taskkill

**Comment by fejta** [comment]
Thank you for creating a SECURITY_CONTACTS file. Please `/close` the issue.
/close

**Pull request merged**